Ports/lua/Lua.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+PORT_DIR=lua
+MAKEOPTS='generic'
+function fetch() {
+    run_fetch_web "http://www.lua.org/ftp/lua-5.3.5.tar.gz"
+    run_patch lua.patch -p1
+    run_patch oslibc.patch -p1
+}
+function configure() {
+    run_export_env CC i686-pc-serenity-gcc
+}
+
+function run_make() {
```

Comment:
This function is already provided by `.port.include.sh` so you shouldn't replace it here.